src/test/kotlin/org/rust/ide/annotator/RsExpressionAnnotatorTest.kt

```diff
@@ -149,6 +149,14 @@ class RsExpressionAnnotatorTest : RsAnnotationTestBase() {
         }
     """)
 
+    fun `test external ABI is unsafe`() = checkErrors("""
+        extern "C" { fn foo(); }
+
+        fn main() {
+            <error>foo()</error>;
```

Comment:
Let's check the description of annotation here too. Just add `descr="some text"` attr into `error` tag

Reply:
Also, it would be great to ensure that `Surround with unsafe block` quick fix works in such case too.
You can find similar tests in `org.rust.ide.annotator.fixes.AddUnsafeTest`